src/app.es6.js

```diff
@@ -36,14 +36,14 @@ class App {
     return co(function * () {
       yield* middleware;
     }).then(function () {
```

Comment:
This could totally be an arrow function, which would implicitly bind this. Right? I think so. Maybe. Anyway, you don't need to change it. Just a thought.